benefits/core/templates/core/agency_index.html

```diff
@@ -7,7 +7,7 @@
 {% block container_content %}
   <h1>{{ page.headline }}</h1>
 
-  <p>{% translate "core.pages.agency_index.h2" %}</p>
+  <h2>{% translate "core.pages.agency_index.h2" %}</h2>
```

Comment:
This will eventually in the #950 PR have a `p-sm` class. This change is not directly part of this issue, but snuck in here. Will eventually be useful for #950

Reply:
950 is already merged... did you mean another PR?

Reply:
oh i meant in the ticket for redesigning the Agency Index. https://github.com/cal-itp/benefits/issues/937

when we fully redesign Agency Index